MastodonSDK/Sources/MastodonUI/View/Content/StatusAuthorView.swift

```diff
@@ -179,7 +179,10 @@ extension StatusAuthorView {
             postActions.append(.editStatus)
         }
 
-        if let statusLanguage = menuContext.statusLanguage, menuContext.isTranslationEnabled {
+        if menuContext.isTranslationEnabled,
```

Comment:
This doesn't look like a fault from this PR, but for some reason, on my self-hosted instance and als mastodon.social for me `menuContext.isTranslationEnabled` is always `false` thus I'm not seeing the menu entry 😕  But if it worked for you then I'm happy to move on. I'll investigate what's wrong on my side then...

Reply:
Thank you for the feedback, I'll have a closer look at it as it happened to me now, too. oO

Reply:
After removing the CoreData-Queue-stuff, it worked for me(tm). Can you give it another testrun, whenever you have a few minutes? That would be great.

Oh, sorry, I meant:

![830jp7](https://github.com/mastodon/mastodon-ios/assets/2580019/17331297-cdda-4831-b56e-764846b851a9)